coalib/output/ConsoleInteraction.py

```diff
@@ -833,8 +847,29 @@ def ask_for_action_and_apply(console_printer,
                                  failed_actions,
                                  result,
                                  file_diff_dict,
-                                 file_dict)
+                                 file_dict,
+                                 applied_actions)
         return ret
+    if action_name == 'ShowAppliedPatchesAction':
+        show_patch_action = ShowPatchAction()
+
+        for key, val in applied_actions.items():
+            this_result = val[0]
+            this_section = val[3]
+            color_res = RESULT_SEVERITY_COLORS[this_result.severity]
+            console_printer.print('\n**** {bear} [Section: {section}] ***'
+                                  '*\n**** Action Applied: {action} ****\n'
+                                  .format(bear=this_result.origin,
+                                          section=this_section.name,
+                                          action=key),
+                                  color=color_res)
+            console_printer.print(format_lines('[Severity: {sev}]'.format(
+                sev=RESULT_SEVERITY.__str__(this_result.severity)), '!'),
+                  color=color_res)
+            show_patch_action.apply_from_section(val[0], val[1], val[2], val[3])
```

Comment:
Same here, usage of those constans wluld be nice
If this happens in the whole file, open an issue for it and make them global constants